src/main/java/net/reikeb/notenoughgamerules/mixin/blocks/AbstractButtonBlockMixin.java

```diff
@@ -0,0 +1,23 @@
+package net.reikeb.notenoughgamerules.mixin.blocks;
+
+import net.minecraft.block.AbstractButtonBlock;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.reikeb.notenoughgamerules.Gamerules;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
```

Comment:
Don't forget to organise imports

Reply:
yes I will do that later when the pr is ready to be merged